src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2TestBase.cs

```diff
@@ -923,11 +928,15 @@ protected async Task SendIncompleteContinuationFrameAsync(int streamId)
             await SendAsync(payload);
         }
 
-        protected Task SendDataAsync(int streamId, Memory<byte> data, bool endStream)
+        protected Task SendDataAsync(int streamId, Memory<byte> data, bool endStream, bool flushFrame = true)
```

Comment:
Nit: is this being used?

Reply:
It was, and then I realized I didn't need to buffer a data frame if I sent it after the un-buffered stream start so I didn't end up using it 🤷‍♀️